bucket/audmes.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "20211104",
+    "description": "AUDio MEasurement System - PC based Oscilloscope and Spectrum analyzer using sound card",
+    "homepage": "https://sourceforge.net/projects/audmes/",
+    "license": "GPLv2",
+    "url": "https://phoenixnap.dl.sourceforge.net/project/audmes/audmes-win32%20binary/2021-11-04/AudMeS-20211104-win32.zip",
+    "hash": "sha256:ca6da925ae4fdd63a2cc0c92901923c2434312a4617170a50db0858960968a73",
+    "bin": "AudMeS-0.1.1-win32/AudMeS.exe",
```

```suggestion
    "pre_install": [
        "Move-Item \"$dir\\AudMeS-*-win32\\*\" \"$dir\\\" | Out-Null",
        "Remove-Item \"$dir\\AudMeS-*-win32\\*\" -Force -Recurse"
    ],
```
We cannot be sure it will always be `AudMeS-0.1.1` in the future. Therefore we need to avoid future errors by the trick. 

A better solution is to find a **release note/documentation** that explains the relation between version strings (`1.1.1`) and release dates (`20211104`). Sadly I couldn't find one.

---

Also, `bin` is not needed if the application does not accept command-line arguments.